internal/git_clone_test.go

```diff
@@ -0,0 +1,67 @@
+package internal
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/databricks/cli/libs/cmdio"
+	"github.com/databricks/cli/libs/git"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestAccGitClone(t *testing.T) {
+	t.Log(GetEnvOrSkipTest(t, "CLOUD_ENV"))
+
+	tmpDir := t.TempDir()
+	cmdIO := cmdio.NewIO("text", os.Stdin, os.Stdout, os.Stderr, "")
+	ctx := cmdio.InContext(context.Background(), cmdIO)
+	var err error
+
+	err = git.Clone(ctx, "https://github.com/databricks/cli.git", tmpDir)
```

Comment:
Using smaller repos for the tests. I am also using repos on my personal account because we should not create new branches in [databricks-empty-ide-project](https://github.com/databricks/databricks-empty-ide-project)